atlantis.env.example

```diff
@@ -0,0 +1,8 @@
+ATLANTIS_GH_USER=<your-gh-user>
+ATLANTIS_GH_TOKEN=<your-gh-access-token>
+ATLANTIS_GH_WEBHOOK_SECRET=<your-gh-webhook-secret>
+ATLANTIS_REPO_ALLOWLIST='github.com/<your-repo>'
+ATLANTIS_REPO_CONFIG=repos.yaml
+ATLANTIS_ENABLE_POLICY_CHECKS=false
+ATLANTIS_LOG_LEVEL=debug
```

Comment:
Seems like we can remove the LOG_LEVEL and I wonder if POLICY_CHECKS should be set to true (so we don't need special instructions for conftest examples)?
```suggestion
ATLANTIS_ENABLE_POLICY_CHECKS=true
```